### PR TITLE
Fix fetch_languages for Bing

### PR DESCRIPTION
**Pull request by MarcAbonce**
## What does this PR do?

**TLDR:**  
 * Bing's issue in #2589 can be solved by hardcoding an Accept-Language header.
 * Startpage didn't return a language list because of a missing HTTP header.
 * Removed articles counter from Wikipedia: https://github.com/searx/searx/pull/2589#issuecomment-782200762

Bing has a list of regions that it supports and some of these regions may have more than one possible language.

In some cases, like Switzerland, these languages are always shown as options, so there is no issue. But in other cases, like Andorra, Bing will only show one language at the time, either the region's default or the request's language if the latter is supported by that region.

For example, if the HTTP request is in French, Andorra will appear as fr-AD but if the same page is requested in any other language Andorra will appear as ca-AD.

This is specially a problem when Bing assumes that the request is in English because it overrides enough language codes to make several major languages like Arabic dissappear from the languages.py file.

To avoid that issue, I set the Accept-Language header to a language that's only supported in one region to hopefully avoid these overrides.

On a separate note, by chance I also found out that Startpage now returns a language list thanks to the User-Agent header.

## How to test this PR locally?

Run `python utils/fetch_languages.py` and verify that languages.py doesn't remove so many languages.
I don't know if searx-bot can be manually triggered to run this branch before merging, just to make sure?

## Related issues

#2589

### CODE REVIEW

**Review by dalf — APPROVED** [review]
On my laptop it works.
On Friday, the github workflow will open a PR, we will be able  to check if it works too.

Thank you !
